run-tests.sh

```diff
@@ -60,7 +60,7 @@ if [ ! -d "$BUILD_FOLDER" ]; then
 	mkdir -p $BUILD_FOLDER
 fi
 
-ERRORTXT=${BUILD_FOLDER}/errors.txt
+ERRORTXT=${SCRIPTDIR}/errors.txt
```

Comment:
Let's write the errors file in the scriptdir instead of in the build folder to avoid that we lose the errors.txt if the build dir is cleaned